src/styles/_catalog.scss

```diff
@@ -1,10 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  grid-template-columns: repeat(4, 200px);
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 200px);
+  }
```

Comment:
You can use a variable here, because card width is often reused here

Reply:
+1, let's create a variable for card width value